xc/xenops_server_xen.ml

```diff
@@ -916,6 +902,35 @@ module Featureset = struct
     |> String.concat ":"
 end
 
+let debug_featuresets xc name featureset default featureset_max index_max =
+  (* For debugging purposes retrieve the default featureset and compare it *)
+  debug "%s Max featureset = %a" name Featureset.pp featureset_max ;
+  let featureset_default = Xenctrl.get_cpu_featureset xc default in
```

Comment:
The caller already has the default featureset to hand.  This is more obvious when the other CPUID debt been dropped.

Reply:
Yes, this can be further simplified now.